cypress/pages/sidebar/main-sidebar.ts

```diff
@@ -2,14 +2,23 @@ import { BasePage } from "../base";
 
 export class MainSidebar extends BasePage {
 	getters = {
-		settings: () => cy.getByTestId('menu-item-settings', { timeout: 5000 }),
+		settings: () => cy.getByTestId('menu-item-settings'),
 		templates: () => cy.getByTestId('menu-item-templates'),
 		workflows: () => cy.getByTestId('menu-item-workflows'),
 		credentials: () => cy.getByTestId('menu-item-credentials'),
 		executions: () => cy.getByTestId('menu-item-executions'),
 	};
 	actions = {
-		goToSettings: () => this.getters.settings().click(),
-		goToCredentials: () => this.getters.credentials().click(),
+		goToSettings: () => {
+			// For some reason Cypress gets confused when clicking on the settings menu item
+			// and it pickups element that will get detached with the sidebar animation
+			// so we need to wait for it to happen first
+			cy.wait(500)
```

Comment:
should these waits happen after page load instead of before every time we want to click these buttons?

Reply:
I'd prefer to have it before clicking on these buttons. Because then you don't need to think about it when writing the spec, otherwise there's a decent chance one might forget to add it and spend time debugging why it doesn't work. This way clicking on side panel would just work, IMO 500ms is worth the trade

Reply:
could really slow tests the more we use this...

Reply:
The root cause of this issue was pending animations when clicking on the sidebar menu item. Should be fixed now so there's no need for cy.wait